example/tcp_connect.cpp

```diff
@@ -0,0 +1,49 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// // Measurement-kit is free software. See AUTHORS and LICENSE for more
+// // information on the copying conditions.
+
+//
+// This example will run a synchronous tcp-connection test
+//
+
+#include <measurement_kit/ooni.hpp>
+#include <iostream>
+#include <string>
+
+int main(int argc, char **argv) {
+    std::string file_name = "";
+    std::string port = "80";
+    std::string name = argv[0];
+    bool verbose = false;
+    char ch;
+    while ((ch = getopt(argc, argv, "f:p:v")) != -1) {
+        switch (ch) {
+        case 'f':
+            file_name = optarg;
+            break;
```

Comment:
This option should not appear, since now the file_name is mandatory and is a positional command line argument.